### PR TITLE
MON-3367: chore: enable delayed-compaction feature flag on Platform and UWM Prometheuses

### PR DESCRIPTION
**Pull request by machine424**
…metheuses

* [ ] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by machine424** [comment]
/retitle MON-3367: chore: enable delayed-compaction feature flag on Platform and UWM Prometheuses

File: jsonnet/components/prometheus.libsonnet [hunk 1]
```diff
@@ -333,6 +333,8 @@ function(params)
         },
       },
       spec+: {
+        // Enable experimental delayed compaction feature
```
Comment (simonpasquier) [inline jsonnet/components/prometheus.libsonnet:336]:
```suggestion
        // Enable experimental delayed compaction feature.
```

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Pull request merged**